src/chapters/exceptions/throw.rst

```diff
@@ -0,0 +1,64 @@
+Throw
+=====
+
+In most programming languages when the compiler, or interpreter encounters code it doesn't know how to handle it has to notify the programmer that something has gone wrong. We call this act "throwing an exception". This is where the term throw comes from.
+
+JavaScript gives us a throw statement to raise exceptions whenever we need to inform the programmer that something happened that the programming language doesn't know how to handle.
+
+JavaScript gives us the ability to raise exceptions using the **throw** statement.
+
+Default Error
+-------------
+
+We can throw a default Error object by using the throw statement, and passing the message in as an argument.
```

Comment:
``throw`` for the second instance. Comma is unnecessary.

Reply:
fixed